images/win/scripts/Installers/Install-Aria2.ps1

```diff
@@ -0,0 +1,8 @@
+################################################################################
```

Comment:
Could you please remove this script and update toolset-2016.json/2019.json files?

Reply:
That didn't exist three weeks ago. I made the changes you requested. Could you take a look again please @al-cheb ?